### PR TITLE
[DBForMySql] Generate MySql

### PR DESCRIPTION
**Pull request by msJinLei**
## Description

The examples and tests are not included temporarily and will be added soon.

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/MySql/readme.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+  - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json
+module-version: 0.1.0
```
Comment (dolauli) [inline src/MySql/readme.md:54]: New RP version is 0.0.1

File: src/MySql/readme.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+title: MySQL
+subject-prefix: 'MySQL'
+
+directive:
+  - from: mysql.json
+    where: $.definitions.VirtualNetworkRule
+    transform: $['required'] = ['properties']
+  - select: command
+    where:
+      verb: Set
```
Comment (dolauli) [inline src/MySql/readme.md:64]: Not sure why need to rename set to update?

File: src/MySql/readme.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    set:
+      verb: Update
+  - where:
+      verb: New$|^Set$|^Remove$|^Get|^Update$|^Invoke$
```
Comment (dolauli) [inline src/MySql/readme.md:68]: There are no Set cmdlets, since they have been renamed to update.

File: src/MySql/readme.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        properties:
+          - Name
+          - StartIPAddress
+          - EndIPAddress
+  - where:
+      parameter-name: StorageProfileBackupRetentionDay
+      subject: Server
+    set:
+      parameter-description: Backup retention days for the server. Day count is between 7 and 35.
+  - from: ServerForCreate.cs
```
Comment (dolauli) [inline src/MySql/readme.md:130]: Shall we need to change the specific file to source-file-csharp? Because it looks not working in some platforms.

**Comment by msJinLei** [comment]
merged by https://github.com/Azure/azure-powershell/pull/11519

**Pull request closed without merging**